modules/core/src/main/java/org/apache/fluo/core/oracle/OracleServer.java

```diff
@@ -299,13 +299,7 @@ private InetSocketAddress startServer() throws TTransportException {
     serverArgs.outputProtocolFactory(new TCompactProtocol.Factory());
     server = new THsHaServer(serverArgs);
 
-    Runnable st = new Runnable() {
-
-      @Override
-      public void run() {
-        server.serve();
-      }
-    };
+    Runnable st = () -> server.serve();
```

Comment:
Could `st` be inlined?

Reply:
I don't see why not. 

Also I left this one since the class is Deprecated. Is that the right approach?
https://github.com/apache/fluo/blob/327ab36c194a73343bcf38b8578b83647979266a/modules/cluster/src/main/java/org/apache/fluo/cluster/runner/AppRunner.java#L160-L166

Reply:
Personally, I would not go looking for improvements to make in deprecated code.  But since you have already found it, I think its worthwhile to make the improvement and make the code more readable.